codechecker_common/source_code_comment_handler.py

```diff
@@ -23,6 +22,22 @@
                         "suppress", "unreviewed"]
 
 
+def contains_codechecker_comment(fp):
+    """Returns true if the file content contains any
+    codechecker review comments.
+    """
+    match = re.search(".*codechecker_.*", fp.read())
+    if not match:
+        return False
+    fp.seek(0)
```

Comment:
I've updated the function and extended the docs so that the position is set back to the original position when the scanning is done (even if the position was not 0)